packages/@okta/vuepress-site/docs/guides/configure-signon-policy/prompt-factor-outside-us/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Prompt for an MFA factor when a user is outside the US
+---
+
+In addition to the Okta Sign-On Policy, there is a sign-on policy for each application that determines the extra levels of authentication that you may want performed before a user can access an application. The following are step-by-step instructions to configure an App sign-on policy to prompt a user for an MFA factor when the user is outside the United States.
+
+> **Note:** This example assumes that you have already [set up a Dynamic Zone](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network) for the United States.
+
+### Select the App
+
+1. From the Developer Console, select **Applications**.
+
+2. Select the app that you want to configure a sign-on policy for. In this example, we are using a Web App.
+
+3. Click **Sign On** and scroll down to the **Sign On Policy** section.
+
+### Create the rule and define conditions
+
+> **Note:** Some options described in this section may not be available in your org. To enable them, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
+
+1. Click **Add Rule**.
+
+2. Enter a **Rule Name**.
+
+3. Select who the rule applies to in the **People** section. In this use case example, select **The following groups and users:**.
+
+4. In the **Groups** box that appears, enter the group that you want this rule to apply to. In this example, we are using **Everyone**. You can also add specific users that you want to include in the **Users** box.
+
+    If you want to exclude specific groups and users from the policy rule, select **Exclude the following users and groups from this rule:** and then specify the groups and users.
+
+5. Specify the location to which you want the policy to apply in the **LOCATION** section of the dialog box. In this example, select **In Zone**.
+
+6. In the **Network Zones** box that appears, enter the name of the zone that you want to apply, and then add it when it appears in the list.
```

Comment:
In keeping with the title of this example, we should instruct the read to select the network zone for the United States here. Right?

Reply:
good catch, thank you